web/massastation/src/pages/Index/Index.tsx

```diff
@@ -69,11 +71,13 @@ export function Index() {
 
   useEffect(() => {
     setRefreshPlugins(refreshPlugins + 1);
-  }, [pluginWalletIsInstalled]);
+  }, [pluginWalletIsInstalled, isLoading]);
 
   function handleInstallPlugin() {
     try {
+      setIsLoading(true);
       mutate({});
+      setIsLoading(false);
```

Comment:
Mutate is async... This should be false on the isSuccess/isError step.

Reply:
done